task_02/danil_dyachenko/02_knight_and_princess/graph_gen.cpp

```diff
@@ -0,0 +1,107 @@
+#include <iostream>
+#include <fstream>
+#include <vector>
+
+struct Vertex{
+    int id;
+    std::vector<int> edges;
+};
+
+struct Edge{
+    int id;
+    std::vector<int> vertices;
```

Comment:
У грани может быть больше 2ух вершин? Зачем здесь использовать вектор, в такм случае лучше использовать массив
```suggestion
    std::array<int, 2> vertices;
```